Client/Frontend/Update/UpdateViewModel.swift

```diff
@@ -5,58 +5,92 @@
 import Foundation
 import Shared
 
-class UpdateViewModel {
-    //  Internal vars
-    var updateCoverSheetModel: UpdateCoverSheetModel?
-    var startBrowsing: (() -> Void)?
+class UpdateViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
 
-    // Constants
-    let updates: [Update] = [Update(updateImage: #imageLiteral(resourceName: "darkModeUpdate"), updateText: "\(String.CoverSheetV22DarkModeTitle)\n\n\(String.CoverSheetV22DarkModeDescription)")]
+    let profile: Profile
+    static let prefsKey: String = PrefsKeys.KeyLastVersionNumber
```

Comment:
q. if the prefs key is in PrefsKeys file then why do we need to make it static? 
Is this only for convenience?

Reply:
yes it's just for convenience, initially I wanted to override the key to use in unit test but at the end I went in other route so I actually don't need this change anymore